qupath-gui-fx/src/main/java/qupath/lib/gui/JavadocViewer.java

```diff
@@ -1,695 +1,110 @@
-/*-
```

Comment:
We should keep the copyright statement in every file that's part of core QuPath. If it has been completely rewritten, then we can set the year to be the current year. Otherwise we can use 2022-2024.

Reply:
Added in last commit